src/components/RedigerAvansertBeregning/RedigerAvansertBeregning.tsx

```diff
@@ -372,26 +518,121 @@ export const RedigerAvansertBeregning: React.FC<{
+                        { ...getFormatMessageValues(intl) }
+                      )
+                    : ''
+                }
+                role="radiogroup"
+                aria-required="true"
+              >
+                <Radio
+                  form={FORM_NAMES.form}
+                  data-testid={`${FORM_NAMES.inntektVsaHeltUttakRadio}-ja`}
+                  value="ja"
+                >
+                  <FormattedMessage id="stegvisning.radio_ja" />
+                </Radio>
+                <Radio
+                  form={FORM_NAMES.form}
+                  data-testid={`${FORM_NAMES.inntektVsaHeltUttakRadio}-nei`}
+                  value="nei"
+                >
+                  <FormattedMessage id="stegvisning.radio_nei" />
+                </Radio>
+              </RadioGroup>
             </div>
           )}
+
+        {localHeltUttak?.uttaksalder?.aar &&
+          localHeltUttak?.uttaksalder?.maaneder !== undefined &&
+          localHarInntektVsaHeltUttakRadio && (
+            <>
+              <div>
+                <TextField
+                  form={FORM_NAMES.form}
+                  name={FORM_NAMES.inntektVsaHeltUttak}
+                  data-testid={FORM_NAMES.inntektVsaHeltUttak}
+                  type="text"
+                  inputMode="numeric"
+                  className={styles.textfield}
+                  label={
+                    <FormattedMessage
+                      id="inntekt.endre_inntekt_vsa_pensjon_modal.textfield.label"
+                      values={{ ...getFormatMessageValues(intl) }}
+                    />
+                  }
+                  description={intl.formatMessage({
+                    id: 'inntekt.endre_inntekt_vsa_pensjon_modal.textfield.description',
+                  })}
+                  error={
+                    validationErrors[FORM_NAMES.inntektVsaHeltUttak]
+                      ? intl.formatMessage(
+                          {
+                            id: validationErrors[
+                              FORM_NAMES.inntektVsaHeltUttak
+                            ],
+                          },
+                          { ...getFormatMessageValues(intl) }
+                        )
+                      : undefined
+                  }
+                  onChange={handleInntektVsaHeltUttakChange}
+                  value={localHeltUttak?.aarligInntektVsaPensjon?.beloep}
+                  max={5}
+                />
+              </div>
+              <div>
+                <AgePicker
+                  form={FORM_NAMES.form}
+                  name={FORM_NAMES.inntektVsaHeltUttakSluttAlder}
+                  label={intl.formatMessage({
+                    id: 'inntekt.endre_inntekt_vsa_pensjon_modal.agepicker.label',
+                  })}
+                  value={localHeltUttak?.aarligInntektVsaPensjon?.sluttAlder}
+                  minAlder={minAlderInntektSluttAlder}
+                  maxAlder={{ aar: 75, maaneder: 11 }}
```

Comment:
Jeg lurer litt på hvorfor akkurat 75+11?

Reply:
Det er fordi man kan tjene opp til alderspensjon tom. året man fyller 75 år.
Dvs. Tilgjengelige aldere kan tidligst starte ved valgt tidspunkt for uttak av 100 % + 1 måned, og max. 75 år og 11 måneder (0–11).

Jeg flytter dette til en konstant i lik linje som andre alders-"grenser". :)